### ISSUE TITLE
Template Card reflow

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#56]
**Pagina:**
[Rijkshuistijl Community Template](https://rijkshuisstijl-community-templates-git-102644-nl-design-system.vercel.app/developer-overheid/homepage)

**Probleembeschrijving:**
Bij inzoomen of in kleine viewports rangschikken de Cards op de pagina zich om minder horizontale ruimte in te nemen, maar ze zijn beperkt tot 2 Cards per rij. De pagina zou in staat moeten zijn om indien nodig slechts één Card per rij te tonen.

**Gerelateerde WCAG:**
WCAG 1.4.10 Reflow

**Referentieafbeelding:**

<img width="510" alt="zoomed in page does not reflow the card components enough so the content is squeezed together in a way the user cannot read the whole content on the screen" src="https://github.com/user-attachments/assets/710c0d80-a6d0-4e73-922c-36d1f79c8d5b" />
